client/resources/sass/site2.scss

```
@@ -462,6 +463,7 @@ form.vertical, .fields-vertical {
         text-align: left;
         padding: 24px 20px;
         font-weight: 500;
+        font-size: 16px;
```

```suggestion
        font-size: 1rem;
```

most of the styles are in rem units. relative to the root element font size.